packages/tenant-process/src/services/readModelService.ts

```diff
@@ -559,6 +560,34 @@ export function readModelServiceBuilder(
       }
       return undefined;
     },
+    async getDelegation(
```

Comment:
maybe it's better `getActiveDelegation` or we could add another parameter for the delegation state